src/flows/components/header/index.tsx

```diff
@@ -72,8 +71,10 @@ const backgroundColor = '#07070E'
 
 type MenuItemType = {
   title: string
-  onClick: () => void
-  icon: IconFont
+  onClick?: () => void
```

Comment:
what is the point of defining an interface is everything is optional

Reply:
Right?

Reply:
rephrased another way:
```
interface ButtonProp {
  menuItems: Array<MenuItemType | MenuDividerType>
}
```